Loop/Managers/Alerts/AlertManager.swift

```diff
@@ -176,6 +177,20 @@ extension AlertManager {
 
 // MARK: Alert storage access
 extension AlertManager {
+    private static func ensureAlertStoreDirectoryExists(_ fileManager: FileManager, _ documentsDirectory: URL?, _ log: DiagnosticLog) {
+        guard let documentsDirectory = documentsDirectory else {
+            // No documents directory means this is running in a unit tests, and store will be in-memory
+            return
+        }
+        do {
+            try fileManager.createDirectory(at: documentsDirectory.appendingPathComponent(AlertStore.storageDirectoryPathComponent),
+                                            withIntermediateDirectories: true,
+                                            attributes: [FileAttributeKey.protectionKey: FileProtectionType.none])
```

Comment:
This changes the dir from class C to class D, which means no protection, even before first unlock. We don't need access before first unlock, so I think this is unnecessary and more open than needed.

Reply:
It's what the other Store's do @ps2 .  See `PersistenceController.swift`, `initializeStack` around line 149.  I'm guessing this is done so it can be accessed BFU.  Why do you say we need no access BFU?

Reply:
iOS should not (according to apple) launch Loop BFU.  Doing so was a bug, apparently. The other inits were written back before we realized that running BFU was not a possibility. So yes, they could be updated.